config/routes.rb

```diff
@@ -29,6 +30,14 @@
       post :select
     end
 
+    #JE Todo: Confirm if we should nest these internal API routes under a namespace like "API::V1".
```

Comment:
Good thought but not needed I think since the API and frontend are in the same codebase.